### ISSUE TITLE
[QUESTION] is buying server files a thing?

### ISSUE THREAD

**Issue by bloomboxes** [GrowtopiaNoobs/GrowtopiaServer#1386]

**Comment by xXAbood** [comment]
u can but i suggest u use Anybody's original source and add your own modifiers.

**Comment by raiterjaki** [comment]
No. You can't earn profits from someone else's work. All the profit should go to the owner (Growtopia Noobs)

**Comment by Bexpg1337** [comment]
if you dont know how to setup you can buy help  but file is free.

**Comment by cmdocmd** [comment]
using original source is better than buying files probably leaked and have alot of bugs in it

**Comment by LaYellow** [comment]
yes if you have money

**Comment by abood1337** [comment]
> u can but i suggest u use Anybody's original source and add your own modifiers.

retard why are you copying me /fp -.- 0.0

**Comment by Hades3377** [comment]
depends, however highly recommended to make your own
